server/main.go

```diff
@@ -39,21 +39,25 @@ var (
 	port       = flag.String("port", ":12345", "Port on which to run the server.")
 )
 
+// Version is the docker image release version number.
+// This should be updated on the release branches whenever a new docker image is built and released.
```

Comment:
Hmm, isn't this the version number for the API that we manually configure (separate from the docker image, which is for the binary releases and should not effect the version number in the response).

Reply:
Updated